### ISSUE TITLE
Filter shot/asset list to only those that have a task assigned to you

### ISSUE THREAD

**Issue by BigRoy** [cgwire/kitsu#263]
### Issue

Our Artists are asking whether it would be possible to easily filter the shot list or asset list to only those assets/shots that they are assigned to.

I referred them to the *My Tasks* page but they felt that is way too cluttered, as it also shows task from other projects. Plus they said it's faster/more useful to quickly filter the shot list as you are browsing it.

Could this be an option?

**Comment by BigRoy** [comment]
This is _somewhat_ related to #244 as both discuss "searchability/readability" of asset/shot-list.

**Comment by flablog** [comment]
Indeed it would be a nice feature. As My Task is note displaying everything (Custom parameters for example or other departments to compare). 
+1

**Comment by frankrousseau** [comment]
Thank you for reporting, but sorry we can't take this one in the current state. "way too cluttered" is  subjective and is not accurate enough to be handled. 
Can you give us too the information they are looking for which is missing from the my tasks page?

**Comment by BigRoy** [comment]
They are looking for only their animation shots for a specific project and alongside be able to see the state of the "previous" task in for that shot (e.g. layout) and investigate potential comments on that previous task if needed.

E.g. a lighting artist might want to see the shots he is assigned to and investigate the state of the animation task prior to it. If it's still WIP he might take a peek in the comments to anticipate what the current state is and what he might need to think about. Plus additionally you might want to also cross-reference the other parameters for each task as would be available in the shot list as @flablog describes: "As *My Task* is not displaying everything".

I think the "clutter" in My Tasks is mostly that it's not filtered to a Project and that sometimes the *Last Comment* can be relatively long having one task take up a lot of screen space vertically. E.g. for me on My Tasks I can currently fit 3-4 tasks on 1920x1080 resolution vertically because the last comments contain a bunch of bullet point lists that make it very long. I like the fact that you can see the last comment, but there might be moment you'd want to hide that too.

Does that help?

**Comment by frankrousseau** [comment]
It's weird that the last comment field is not shorten. Can you send me a screen capture?

**Comment by BigRoy** [comment]
![bulletpoints](https://user-images.githubusercontent.com/2439881/62467740-c0406800-b794-11e9-8b0b-e8d4fe73c46f.png)

I think it's just that a list of bullet points is not shortened? (This is just a small test list)

**Comment by frankrousseau** [comment]
Ok thank you, I see what's wrong now

**Comment by frankrousseau** [comment]
What about removing the last comment column and replacing it by the last comment date? It was set there before the side panel was added. Not sure it's still useful now.

**Comment by flablog** [comment]
I think it's more usefull to *see* it. A date might mean nothing for a graphists with dozens of open tasks

**Comment by BigRoy** [comment]
> What about removing the last comment column and replacing it by the last comment date? It was set there before the side panel was added. Not sure it's still useful now.

I agree with _flablog_ that it is useful somewhat to get a hint of what happened in the comment.

How about just trimming to a certain maximum amount of characters **and** no more than max. 2 new lines? 

This long test comment would then become:

```
Test comment
- with
...
```

> the last comment date? 

The date however could be useful data too e.g. as separate column, but I'm not sure it's needed. ;)
